src/main/java/graphql/execution/FetchedValue.java

```diff
@@ -45,64 +38,13 @@ public Object getLocalContext() {
         return localContext;
     }
 
-    public FetchedValue transform(Consumer<Builder> builderConsumer) {
-        Builder builder = newFetchedValue(this);
-        builderConsumer.accept(builder);
-        return builder.build();
-    }
-
     @Override
     public String toString() {
         return "FetchedValue{" +
                 "fetchedValue=" + fetchedValue +
-                ", rawFetchedValue=" + rawFetchedValue +
                 ", localContext=" + localContext +
                 ", errors=" + errors +
                 '}';
     }
 
-    public static Builder newFetchedValue() {
```

Comment:
this is a breaking change becauwe in theory some one could have used these builder methods - BUT... we never accept them - the engine is the only code that creates them - people can consume them sure - but we dont expect them to build them